app/src/main/res/layout/activity_crime_pager.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    >
+
+    <android.support.v4.view.ViewPager
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/crimeViewPager"
+        android:layout_width="match_parent"
+        android:layout_height="464dp"
```

Comment:
Did you test in small screens? Like a 3.7 or 4 inch screen. Maybe you could use the weight atribute. Thoughts @matir91 ?